db/migrate_cards/20160106014913_import_company_topic_right_sidebar_structure.rb

```diff
@@ -0,0 +1,8 @@
+# -*- encoding : utf-8 -*-
+
+class ImportCompanyTopicRightSidebarStructure < Card::Migration
+  def up
+    import_json "company_topic_right_sidebar_structure.json"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.